src/ui/react/test/button-link-edit.jsx

```diff
@@ -41,6 +41,31 @@
             assert.strictEqual('<p><a href="http://link.com" target="">selection</a></p>', data);
         });
 
+        it.only('should create link with default http protocol after editiing', function() {
```

Comment:
Also, maybe change the message a little bit... `should use the default protocol when editing an existing link`